File: Data Structures & Algorithms/duplicate-integer/submission-1.swift
class Solution {
    func hasDuplicate(_ nums: [Int]) -> Bool {
        var hashMap = [Int : Int]()

        for i in nums {
            hashMap[i, default: 0] += 1
        }

        for (key, value) in hashMap {
            if value > 1 { return true }
        }
        return false
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        # Create a hashmap
        hashMap: dict[int, int] = {}

        #Loop through the array.
        for index in range(len(nums)):
            # Check the complement value.
            complement = target - nums[index]

            if complement in hashMap.keys():
                return [hashMap[complement], index]
            else:
                hashMap[nums[index]] = index

        return [-1, -1]

        